### PR TITLE
[ROCm] Adding rocm_py3x_pip.sh scripts for building TF+ROCm whls

### PR DESCRIPTION
**Pull request by deven-amd**
/cc @angerson @perfinion

### CODE REVIEW

**Comment by angerson** [comment]
Thanks for working on this!

However, this directory is pretty tightly restricted to just the scripts that the TensorFlow team maintains. Would you mind putting these scripts in the SIG Build repository instead?

**Comment by deven-amd** [comment]
> Thanks for working on this!
> 
> However, this directory is pretty tightly restricted to just the scripts that the TensorFlow team maintains. Would you mind putting these scripts in the SIG Build repository instead?

The problem with sticking this in the SIG Build repo, is that these scripts become decoupled from the scripts they call (for eg `pip_new.sh` and others), from a version control point of view. These scripts do need tweaking from time to time, and those tweaks are typically tied to changes in the TF repo.

For e.g. imagine if we need to make some change to these scripts after we cut the TF 2.6 branch (as a consequence of some other change in the TF repo, which also occurs after we cut the TF 2.6 branch). At that point, we will need two versions of these scripts too (one that is tied to the TF 2.6 branch and does not have the changes and another in the master branch which has the changes).

So in short, it would be preferable to keep these scripts in the TF repo itself. How about here? - 

https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/ci_build/linux/rocm

**Comment by deven-amd** [comment]
> Good point. tools/ci_build/linux/rocm would be a better place for these than tensorflow/build.

@angerson done. please re-review

**Review by angerson — APPROVED** [review]

**Review by gbaned — APPROVED** [review]
